scripts/cosmos/.gaiad/config/addrbook.json

```diff
@@ -1,4 +1,4 @@
 {
-  "key": "84524d4c173831437f628f5d",
-  "addrs": []
-}
+	"key": "84524d4c173831437f628f5d",
+	"addrs": []
+}
```

Comment:
Why does this use tabs and missing trailing newline now?

Reply:
I didn't touch this myself so I think the gaia application must have done this.

Reply:
Confirmed: starting the application reformats this file.